===================================================================
RCS file: /cvs/gstreamer/gstreamer/libs/gst/dataprotocol/dataprotocol.c,v
retrieving revision 1.7
retrieving revision 1.8
diff -u -r1.7 -r1.8
--- dataprotocol.c	2004/10/01 16:49:01	1.7
+++ dataprotocol.c	2005/02/18 13:58:36	1.8
@@ -83,7 +83,7 @@
 {
   int i;
   int n = 8;                    /* number of bytes per line */
-  gchar *line = g_malloc (3 * n);
+  gchar *line = g_malloc (3 * n + 1);
 
   GST_LOG ("dumping byte array of length %d", length);
   for (i = 0; i < length; ++i) {
